WMF Framework/NavigationBar.swift

```diff
@@ -17,7 +17,16 @@ public class NavigationBar: SetupView, FakeProgressReceiving, FakeProgressDelega
     fileprivate let backgroundView: UIView = UIView()
     public var underBarViewPercentHiddenForShowingTitle: CGFloat?
     public var title: String?
-    
+
+    public init() {
+        super.init(frame: CGRect(x: 0, y: 0, width: 320, height: 44))
+        titleBar.frame = bounds
```

Comment:
This won't be set if `init(frame:` is called on `NavigationBar`. Since `init(frame:` is the designated initializer for `UIView`, `init` should be a convenience `init` that calls `self.init(frame:` with the default value

Reply:
haha was just updating that 😂